repository/mysql/schedules/record.go

```diff
@@ -13,14 +14,24 @@ type Schedule struct {
 	Updated_at    time.Time
 }
 
-type Class struct {
-	Id          int `gorm:"primaryKey"`
-	Name        string
-	Description string
-	Picture_url string
-	Category    string
-	Status      string
-	Schedules   []Schedule `gorm:"many2many:class_schedules"`
-	Created_at  time.Time
-	Updated_at  time.Time
+func fromDomain(domain schedules.Domain) Schedule {
```

Comment:
The function name should be written in PascalCase so the function can be imported to other packages